README.md

```diff
@@ -113,8 +132,13 @@ Your query here
 
 5. How many offices are in the state of Colorado (United States).
 
-```
-Your query here
+```sql
+SELECT count(*) AS list_of_office
+FROM offices AS o
+         INNER JOIN states AS s
+                    ON o.state_id = s.id
+WHERE s.name = 'Colorado'
+  AND s.country_id = (SELECT id FROM countries AS c WHERE name = 'United States');
```

Comment:
I was not completely sure how to solve it, so I tried doing it in the first way that I realized worked. Now, I have a better understanding of how to use `JOINS ` in queries, and it would be more readable to use an `INNER JOIN` instead of a subquery